aw_watcher_afk/config.py

```diff
@@ -3,13 +3,13 @@
 
 default_settings = {
     "timeout": "180",
-    "update_interval": "30",
```

Comment:
Ah, alright. I would prefer interval on both instead of time on both, but it doesn't make that much of a difference so it's fine.